misk/src/main/kotlin/misk/clustering/fake/ExplicitClusterResourceMapper.kt

```diff
@@ -30,6 +31,9 @@ class ExplicitClusterResourceMapper : ClusterResourceMapper {
   }
 
   override fun get(resourceId: String): Cluster.Member {
+    if (mappings.isEmpty()) {
+      throw NoMembersAvailableException(resourceId)
```

Comment:
Didn't know if this should just return null or throw exception. Throwing exception seemed better here since Kotlin doesn't like nulls

Reply:
Agree, it's better to have an explicit exception vs a null which most callers would just convert into a generic and uninformative NPE